runtime/Makefile

```diff
@@ -0,0 +1,152 @@
+##
+#
+# Licensed under the Apache-2.0 license.
+#
+# File Name:
+#
+#    Makefile
+#
+# Abstract:
+#
+#    File contains helper routines for building debugging Caliptra runtime
+#    firmware.
+#
+##
+
+RUSTFLAGS=RUSTFLAGS=-Ctarget-feature=+relax
+TARGET_DIR=../target/riscv32imc-unknown-none-elf/firmware
+CURRENT_DIR = $(shell pwd)
+GIT_REV = $(shell git rev-parse HEAD)
+
+default: build
+
+build:
+	$(RUSTFLAGS) cargo build \
+		--profile firmware \
+		--target=riscv32imc-unknown-none-elf \
+		--no-default-features \
+		--features riscv
+
+build-emu:
```

Comment:
I'm skeptical this target does anything useful...

Reply:
Ya you're right, cargo run should just build it if necessary